src/test/java/com/sprint/mission/discodeit/storage/S3BinaryContentStorageTest.java

```diff
@@ -0,0 +1,51 @@
+package com.sprint.mission.discodeit.storage;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.UUID;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+
+
+@SpringBootTest
+@ActiveProfiles("test")
+class S3BinaryContentStorageTest {
```

Comment:
나중에 CI에서 자동으로 테스트 돌릴 때 실제 S3에 업로드가 이루어질 수 없으니 `@Disabled` 애노테이션 붙여도 좋을 것 같아요.  
```suggestion
@Disabled("수동으로 실행")
class S3BinaryContentStorageTest {
```